Examples/Application Owner/README.md

```diff
@@ -0,0 +1,44 @@
+# Send Apps to App Owner
+
+The purpose of this script is to automate the process of exporting applications / users from DPC for use in Application Owner.
+
+## Requirements
+
+To be able to perform the export / import process you will require API keys for both DPC and AOM.
+
+## Limits
+
+The import process is limited to a maximum of 10,000 applications in a single batch.  If there are more than 10,000 applications in your DPC instance you will need to process them in batches.
+
+If you only have a trial license for AOM then you are limited to a maximum of 100 users.  If an export / import process increases the user count to the limit then it will be aborted preventing any more applications being imported.
+
+### Usage
+
+The script `SendAppsToAppOwner.ps1` should be executed with the following mandatory parameters:
+
+| Parameter | Description | Example Value |
+|-----------|-------------|---------------|
+| DpcInstance | The DPC instance to export from | https://master.internal.juriba.com:8443 |
+| DpcApiKey | A valid API key for your DPC instance | pCyty***** |
+| AomInstance | The AOM instance to import to | https://ao-uat.eu.juriba.app |
+| AomApiKey | A valid API key for your AOM tenant | hoF7k***** |
+
+The following optional parameters can also be provided:
+
+| Parameter | Description | Default if not provided | Validation |
+|-----------|-------------|-------------------------|------------|
+| InputBatchLength | The maximum number of records returned in a single call to DPC | 10,000 | Must be between 1 & 10,000 |
+| InputBatchStartOffset | The offset within the initial batch to start from, can be used to skip previously processed data | 0 | Must be >= 0 | 
+| InputBatchLimit | The number of applications to process | 0 | Must be >= 0 and no less than `InputBatchStartOffset` |
```

Comment:
I wonder the max number of apps to in import is an easier concept than the end index

Reply:
I've refactored the script so that it is a maximum.